scripts/dts/gen_defines.py

```diff
@@ -488,6 +488,8 @@ def controller_and_data_macros(entry, i, macro):
     ret[f"{macro}_IDX_{i}_NAME"] = quote_str(entry.name)
     # DT_N_<node-id>_P_<prop-id>_NAME_<NAME>_PH
     ret[f"{macro}_NAME_{name}_PH"] = f"DT_{entry.controller.z_path_id}"
+    # DT_N_<node-id>_P_<prop-id>_NAME_<NAME>_EXISTS
```

Comment:
Could you please update macros.bnf in the docs too?

Reply:
Totally forgotten this doc. Will update as well. Thanks

Reply:
Done, Please suggest is not correct.

Reply:
This part looks fine. I see now that it's covered by the existing `prop-suf`:

```
prop-suf = 1*( "_" gen-name ["_" dt-name] )
```

In this case, `prop-suf` has to match `_NAME_<NAME>_EXISTS`, which it does, basically as a gen-name + dt-name + gen-name sequence.

Reply:
So do I need an additional update?